src/Rules/Documentation.hs

```diff
@@ -10,6 +10,81 @@ import Settings
 import Target
 import Utilities
 
+docPackage :: Package
+docPackage = hsLibrary "Documentation" "docs"
```

Comment:
Not sure yet what to do with this fake package. We might add a `TODO` asking to fix this hack later.

Probably, the right solution is to add a new package type, but that can be done separately.